purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/PaywallAnimation.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.paywalls.components
+
+import com.revenuecat.purchases.InternalRevenueCatAPI
+import dev.drewhamilton.poko.Poko
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonClassDiscriminator
+
+@Suppress("unused")
+@InternalRevenueCatAPI
+@Poko
+@Serializable
+@SerialName("animation")
+class PaywallAnimation(
+    @get:JvmSynthetic val type: AnimationType,
+    @get:JvmSynthetic
+    @SerialName("ms_delay")
+    val msDelay: Int?,
+    @get:JvmSynthetic
+    @SerialName("ms_duration")
+    val msDuration: Int?,
+) {
+
+    @OptIn(ExperimentalSerializationApi::class)
+    @InternalRevenueCatAPI
+    @Serializable
+    @JsonClassDiscriminator("type")
+    sealed class AnimationType {
```

Comment:
Same for sealed classes... We shoudl write a custom deserializer so we pick a default in case new values are added later on.